STYLE-GUIDE.md

```diff
@@ -2,22 +2,41 @@
 
 This document acts as a Style Guide for the language and wording used in exercises.
 
-## General principle
+## Application
 
-All content should be written in US English.
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
+
+### Measurements
 All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
-In the future other translations may occur, but the "official" Exercism language is US English.
 
-## Explain or substitute mathematical and esoteric terms.
+### Abbreviations, acronyms and initialisms
+> All abbreviations, acronyms and initialisms should be defined. 
```

Comment:
Could we use `<abbr>` (and perhaps `<dfn>`) for this? That way people who know what it means can ignore it, but those that don't can look up the meaning. It avoids cluttering up the text and given that it's part of HTML, it'll be accessible too.

E.g. `<dfn id="html"><abbr title="HyperText Markup Language">HTML</abbr>
</dfn> is a markup language used to create the semantics and structure
of a web page.` (from https://developer.mozilla.org/de/docs/Web/HTML/Element/abbr#Eine_Abk%C3%BCrzung_definieren)

Reply:
Personally, I don't really like the hover-to-show definition user experience and prefer just spelling it out in the prose.

Reply:
(could be a user setting, expand by default)